modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala

```diff
@@ -560,11 +565,17 @@ object ObservationService {
           )
         }
 
+        def itcValidations(params: GeneratorParams): F[ObservationValidationMap] =
+          itcService(itcClient).selectOne(pid, oid, params).map:
+            // N.B. there will soon be more cases here
+            case Some(_) => ObservationValidationMap.empty
+            case None => ObservationValidationMap.empty.add(ObservationValidation.itc("ITC results are not present."))
 
         for {
-          genVals <- generatorValidations
-          cfpVals <- cfpValidations
-        } yield (genVals |+| cfpVals).toList
+          (genVals, op) <- generatorValidations
+          cfpVals       <- cfpValidations
+          itcVals       <- op.filter(_ => cfpVals.isEmpty).foldMapM(itcValidations) // only compute this if cfp and gen are ok
```

Comment:
I'm only computing the ITC validation if everything else is ok; there's no point complaining about ITC if the asterism or config are invalid.